chain/src/block.rs

```diff
@@ -68,6 +68,13 @@ mod tests {
 				"000000e869e3a0fa79858a51b4b1d09a6480dcdb37bae63653fcb11a718abf3f",
 				"6476c40e8de0771a76db44e1d9b3f0a79bd0307e0174245af5b8923a1142f712",
+			),
+			(
+				// https://zcash.blockexplorer.com/block/00000000007ef95f986ed8309d0ed6a1b6174c90b9c7f4d0dfc40f7147315e79
```

Comment:
had problems retrieving tx using `rawtx` API => added this test to make sure we deserialize/serialize block correctly